### ISSUE TITLE
.size() for an .svg image throws an error

### ISSUE THREAD

**Issue by SamVerschueren** [aheckmann/gm#486]
Retrieving the size of a `png` image for example works correctly.

```js
gm('icon.png').size((err, size) => {
    console.log(size);
    //=> {width: 200, height: 200}
});
```

When providing an `svg` image, this is not working.

```js
gm('icon.svg').size((err, size) => {
    console.log(err.message);
    //=> 'Command failed: gm identify: Request did not return an image.'
});
```

##### temporary workaround

```js
gm('icon.svg').identify((err, identity) =>  {
    console.log(identity.size);
    //=> {width: 200, height: 200}
});
```

**Comment by adurrive** [comment]
I'm not sure if it is an issue, a feature request, or something else. But with svg, you need to use the -verbose flag with identify to get any information:
```
gm identify -verbose icon.svg
```
Without it, GraphicsMagick just reads the headers of the file and that doesn't work with svg. Back to the gm module, identify() uses the -verbose flag while size() doesn't have it, which explains the error.

**Comment by SamVerschueren** [comment]
It was meant as issue :).

So basically, this means that by adding `verbose: true` to [this](https://github.com/aheckmann/gm/blob/master/lib/getters.js#L28) line should solve this?

**Comment by adurrive** [comment]
Yep but then you're removing one of the appeal of size() compared to identify(), which is speed ;).

I've altered my pull request to make the file extension available throughout the module, and added a quick custom fix for this issue. We'll see if it gets through.
